explainer-server/app/views/explainEditor.scala.html

```diff
@@ -68,6 +66,24 @@
             }
         }
 
+        /*
+         * This function (and the accompanying handler) exists because there doesn't seem to exist a way
+         * to generate a checkbox using ScalaTags (ScalaJS's HTML generation library)
+         */
+        function installDisplayTypeTickBox(displayType){
+            console.log(displayType);
+            var checkedstring = displayType=='Expandable' ? ' checked ' : ''
+            $(".explainer-editor__displayType-inner").html('<input class="explainer-editor__displayType-checkbox" type="checkbox" name="displayType" value="long" '+checkedstring+'> Long explainer')

```

Comment:
@philmcmahon Scala tags would be best if possible, but, if not, there's a more JS-y way of doing this using the DOM api ([this sort of stuff](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement))

Also, for your second one, IDs in HTML just lead to pain and couples the JS a bit too tightly to the HTML without any real benefits I feel. I like [data attributes](https://developer.mozilla.org/en/docs/Web/Guide/HTML/Using_data_attributes) or, failing that, a class as the JS shouldn't care whether it's unique or not.